src/render/Renderer.cpp

```diff
@@ -1757,6 +1767,13 @@ void CHyprRenderer::arrangeLayersForMonitor(const MONITORID& monitor) {
 
     CBox usableArea = {PMONITOR->vecPosition.x, PMONITOR->vecPosition.y, PMONITOR->vecSize.x, PMONITOR->vecSize.y};
 
+    if (g_pHyprError->active() && g_pCompositor->m_pLastMonitor == PMONITOR->self) {
+        const auto HEIGHT              = g_pHyprError->height();
+        PMONITOR->vecReservedTopLeft.y = HEIGHT;
```

Comment:
I didn't test it but I assume because of this line the reserved space will always be at the top, even if the user chose to move their error bar to the bottom, as implemented in https://github.com/hyprwm/Hyprland/pull/6111 ?

Reply:
oh, I forgot about that existing.

Reply:
done